exercises/practice/sum-of-multiples/runtests.jl

```diff
@@ -0,0 +1,66 @@
+include("sum-of-multiples.jl")
+using Test
+
+@testset "no multiples within limit" begin
+    @test sum_of_multiples(1, [3, 5]) == 0
+ end
+
+@testset "test one factor has multiples within limit" begin
+    @test sum_of_multiples(4, [3, 5]) == 3
+end
+
+@testset "more than one multiple within limit" begin
+    @test sum_of_multiples(7, [3]) == 9
+end
+
+@testset "more than one factor with multiples within limit" begin
+    @test sum_of_multiples(10, [3, 5]) == 23
+end
+
+@testset "each multiple is only counted once" begin
+    @test sum_of_multiples(100, [3, 5]) == 2318
+end
+
+@testset "a much larger limit" begin
+    @test sum_of_multiples(1000, [3, 5]) == 233168
+end
+
+@testset "three factors" begin
+    @test sum_of_multiples(20, [7, 13, 17]) == 51
+end
+
+@testset "factors not relatively prime" begin
```

Comment:
This exercise, and largest-series-product, are being discussed as potential big-5 challenges for [Analytical April](https://forum.exercism.org/t/please-help-choose-featured-exercises-for-12in23/4242/11). Both are in Python and R but not Julia. I said I'd try to add them to Julia, despite being a novice at this sort of thing, so that Erik & Jeremy have options when they make the final selection.

One way forward is that I make the Julia changes you suggest and see what you think. Are you OK with treating this as an investment for the future? I'll try to learn, then contribute enough in future to provide a return on investment.

Also, I'll go and read more about Julia, instead of submitting Python with slightly altered punctuation. I hang out with astrophysicists these days, so (as an aside) I'd be interested to see the Professor's eyebrows rise when I submit (graduate level) homework assignments in Julia instead of Python or C. This language deserves to be more widely adopted.

Apologies for dragging you into this, but I have to go through a learning process. If we get this up with just "moderate personal embarrassment, nobody died" I'll count that as a result.

Incidentally, the intro, blurb and tests are from the standard problem specifications [provided by @kytrinyx](https://github.com/exercism/problem-specifications/tree/main/exercises/sum-of-multiples), and I copied a lot of this from the Python track: it's not my wording. I agree your suggestions are mostly better (maybe not coprime - I did enough college maths to understand, but some students could be confused).

It's evening here, I'll work on this on Monday.

_P.S. Don't be fooled by my Arizona location. I grew up in Prescot, about 30 miles west of you._

Reply:
There's no need to apologise! I hope my tone wasn't unfriendly or unwelcoming!

Thanks for this PR. I'll open a new PR on the problem-specifications repo with some changes to the instructions. If you have any ideas about how to describe the problem then they will be very welcome.

As for this PR, let's leave the test descriptions as-is and we just need to write the approaches file and maybe change the introduction text (we can synchronise it again with problem-specifications if the community agrees on a new intro, but it's okay if this track has a slightly different text until then).

I'll write an approaches file shortly, please feel free to make changes to the example.jl or any other part :)

Reply:
In the spirit of optimistic merging, I'll merge it as is and the text changes and approaches can be done in a follow-up PR